lib/libmbedtls/core/tee_lmd_provider.c

```diff
@@ -75,6 +85,110 @@ static int mbd_rand(void *rng_state, unsigned char *output, size_t len)
 	* defined(CFG_MBEDTLS_HMAC_PRNG)
 	*/
 
+#if defined(_CFG_CRYPTO_WITH_CIPHER) || defined(_CFG_CRYPTO_WITH_MAC) || \
+	defined(_CFG_CRYPTO_WITH_AUTHENC)
+/*
+ * Get the Mbedtls chipher info given a TEE Algorithm "algo"
+ * Return
+ * - TEE_SUCCESS in case of success,
+ * - NULL in case of error
+ */
+static const mbedtls_cipher_info_t *get_cipher_info(uint32_t algo,
+						size_t key_len)
+{
+	/* Only support key_length is 128 bits of AES in Optee_os */
```

Comment:
I can't make sense of this comment.

Reply:
This comment should be deleted. I will fix it.